std/root.ln

```diff
@@ -321,14 +321,38 @@ export fn push(arr: Array<bool>, val: bool) {
   return arr
 }
 export fn pop(arr: Array<any>) = poparr(arr)
-export each // opcode with signature `fn each(arr: Array<any>, cb: function): void`
+export each // parallel opcode with signature `fn each(arr: Array<any>, cb: function): void`
+export fn eachLin(arr: Array<any>, cb: function): void = eachl(arr, cb)
 export map // parallel opcode with signature `fn map(arr: Array<any>, cb: function): Array<any>`
-export mapl // serial opcode with signature `fn mapl(arr: Array<any>, cb: function): Array<any>`
-export reduce // opcode with signature `fn reduce(arr: Array<any>, cb: function): any`
+export fn mapLin(arr: Array<any>, cb: function): Array<anythingElse> = mapl(arr, cb)
+export fn reduce(arr: Array<any>, cb: function): any = reducel(arr, cb)
+export fn reducePar(arr: Array<any>, cb: function): any = reducep(arr, cb)
+type InitialReduce<T, U> {
```

Comment:
let's leave a comment explaining the different types of reduce and reducePar signatures?

Reply:
Sounds good.